docs/Process.md

```diff
@@ -121,3 +121,61 @@ When filing issues relating to a currently in-progress proposal (i.e. any propos
 
 When filing issues relating to a completed feature or specification document
 please use the _Spec_ template.
+
+## Fast-Track for Extensions
+
+Some features for HLSL expose new hardware capabilities and require years of
+development before they can be made public. For these features we have a
+fast-track process to incorporate platform-specific and vendor-specific
+extensions to HLSL as long as they are _conforming extensions_. Extension
+proposals should use the [Extension
+Template](/proposals/templates/extension-template.md).
+
+### Conforming Extensions
+
+Conforming extensions are language features which do not add new language
+behavior or syntax. They cannot remove or deprecate functionality, and they
+cannot be breaking changes. They can add new builtin function declarations,
+builtin data types, and attributes as long as the added features do not change
+the rules of the language. All added declarations must be under a namespace, and
+cannot be under the `hlsl`, `std` or global namespaces which are reserved for
+core language features.
+
+#### Extension Attribute Restrictions
+
+Additionally there are limitations specifically for attributes. Added attributes
+may not modify canonical types, or otherwise change how HLSL code is
+interpreted. They can compile down to metadata that produces annotations, and
+they can be used for analysis and verification. Adding attributes that change
+language behavior must be done through the full review process. As some concrete
+examples:
+
+* Type attributes like `precise` and `groupshared` _can not_ be added as
+  extension attributes, because they (1) modify underlying canonical types, and
+  (2) are not namespaced.
+* Type attributes like `[vk::ext_reference]` _can not_ be added as extension
+  attributes, because it modifies the underlying canonical type.
+* Parameter attributes like `[vk::ext_literal]` _can_ be added as an extension
+  attribute, because it does not modify the type or behavior of the language it
+  just annotates a declaration for additional verification.
+* Entry attributes like `[NodeIsProgramEntry]` _can_ be added as an extension
+  attribute, because it does not modify the code generation of the function it
+  only changes generated metadata.

```

Comment:
Technically `[NodeIsProgramEntry]` wouldn't be valid since it isn't namespaced, but I still think it's a good example. Can we word this to reinforce that?